include/vcpkg/base/parse.h

```diff
@@ -137,10 +133,10 @@ namespace vcpkg
         void add_warning(LocalizedString&& message, const SourceLoc& loc);
 
         const ParseError* get_error() const { return m_messages.error.get(); }
-        std::unique_ptr<ParseError> extract_error() { return std::move(m_messages.error); }
+        std::unique_ptr<ParseError> extract_error() && { return std::move(m_messages.error); }
```

Comment:
These are intentionally non-rvalue - `extract_blah` in vcpkg means "do a move inside" (see also `LocalizedString::extract_data()`)

Reply:
So reverted.